### ISSUE TITLE
Guiding Principles

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/scholarly-html#16]
With thanks to @dauwhe (schema:name: Dave Cramer, since I don't think he's officially in this repo for some reason) for highlighting this issue in another project. Perhaps we need to step out of the weeds and understand, as a group, what our guiding principles are. HTML has a [document](https://www.w3.org/TR/html-design-principles/) that outlines this in more detail than necessary for this group. Note especially, section 3.2 "Priority of Constituencies"
"In case of conflict, consider users over authors over implementors over specifiers over theoretical purity. In other words costs or difficulties to the user should be given more weight than costs to authors; which in turn should be given more weight than costs to implementors; which should be given more weight than costs to authors of the spec itself, which should be given more weight than those proposing changes for theoretical reasons alone. Of course, it is preferred to make things better for multiple constituencies at once." 
In other words, users first, then sort our the back end. So, who are the users? 
1. Human authors 
2. Human tool developers 
3. Processors (not really, they will do what we tell them as long as stuff works)
4. ???

**Comment by essepuntato** [comment]
I think there was a thread in December similar to this in the SH mailing list:

https://lists.w3.org/Archives/Public/public-scholarlyhtml/2015Dec/0001.html

There is also a page on this repo exemplifying the discussion in that thread: 

https://github.com/w3c/scholarly-html/blob/gh-pages/goals.md

I think both may be useful to this issue.

**Comment by TzviyaSiegman** [comment]
@essepuntato Thank you for the reminder. There is a bit of difference between goals and design principles. I am suggesting agreeing to some design principles (and especially user base) so that we have guidance to reach our goals. Groups always encounter conflicts when writing specs. Preparing guiding principles offers us a method for resolving them.

**Comment by essepuntato** [comment]
@TzviyaSiegman,

Yes of course, they are different indeed, but even interlinked somehow I think. That's why I've referenced them here :-)
